ui/packages/docs/docs/reference/config.md

```diff
@@ -150,11 +150,30 @@ target lvl=88 resist=0.1 pos=0,0 radius=2 freeze_resist=0.8 hp=9999 particle_thr
 | `particle_threshold` | Only available if the `hp` is set. Determines after how much damage the enemy drops elemental particles. Example: If the enemy has 500 HP and this is set to 200, then the enemy will drop particles at 300 and 100 HP. | - |
 | `particle_drop_count` | Only available if the `hp` is set. Number of elemental particles to drop at `particle_threshold`. | - |
 | `particle_element` | Only available if the `hp` is set. Element of the particle at `particle_threshold`. Defaults to clear if not set. | - |
+| `type` | Sets the stats of the in-game enemy by name. Overwrite HP and resistances of the target. | - |
 
 :::danger
 All configs must have at least one enemy specified. Otherwise you will get an error. 
 :::
 
+:::info
+To use custom HP or resistances with `type`, include them after `type`.
+Example:
+```
+target lvl=100 type=eremitegalehunter[mult=1] radius=2 pos=0,2.4 hp=10000;
+```
+:::
+
+:::info
+Optimal params may be added to the target type via `[]` after the enemy name.
+:::
+
+#### Optional target type params
+| name | description | default |
+| --- | --- | --- |
+| `hp_mult` | HP multiplier for the enemy. The default is the Spiral Abyss multiplier. | 2.5 |
+| `particles` | Use particle theresholds from the enemy. Use `0` if you want to use a custom threshold (`pacticle_threshold`). | 1 |
```

Comment:
Btw, are you sure that you need this particles param? Shouldn't it just default to using the particle_threshold, if it is defined, over enemy particles (allowing overwrite by just defining particle_threshold instead of having to specify particle_threshold and particles=1)? Once this comment is resolved, I think you can mark syntax as finalized in the PR TODO (could also ask srl for his opinion on `target ... type=<name>[hp_mult=...]`)

Reply:
if the user doesn't want the energy generation from either `particle_threshold` or an enemy curve? e.g. use `every energy` instead of them? then they need to use `particle_threshold=99999999 particle_drop_count=0`?

Reply:
Ok, then leave as is.